VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -1147,6 +1151,13 @@ export abstract class WaitingRoomBaseDirective {
         return this.participant.role === Role.Judge || this.participant.role === Role.StaffMember;
     }
 
+    isAnyHostInHearing(participant: ParticipantResponse) {
+        return (
+            (participant.role === Role.Judge && participant.status === ParticipantStatus.InHearing) ||
+            (participant.role === Role.StaffMember && participant.status === ParticipantStatus.InHearing)
```

Comment:
(participant.role === Role.Judge || participant.role === Role.StaffMember) && participant.status === ParticipantStatus.InHearing
```suggestion
           (participant.role === Role.Judge || participant.role === Role.StaffMember) && participant.status === ParticipantStatus.InHearing
```